src/Video/DebugDrawing.cpp

```diff
@@ -199,6 +229,19 @@ void DebugDrawing::CreateVertexArray(const glm::vec3* positions, unsigned int po
     glBindVertexArray(0);
 }
 
+void DebugDrawing::CreateCircle(glm::vec3*& positions, unsigned int& vertexCount, unsigned int detail) {
+    vertexCount = detail * 2;
+    positions = new glm::vec3[vertexCount];
+    
+    unsigned int i = 0;
+    for (unsigned int l = 0; l <= detail; ++l) {
+        float angle = static_cast<float>(l) / detail * 2.0f * glm::pi<float>();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') identifier should not be composed of only 'l' and 'O' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT010)